Makefile

```diff
@@ -3,16 +3,13 @@ PROJECT = rovers
 COMMANDS = .
 
 GO_BUILD_ENV = CGO_ENABLED=0
+PKG_OS = linux darwin windows
```

Comment:
Do we still want to build for windows? None of the rest of the projects are doing it

cc @ajnavarro @smola

Reply:
Yes. Other projects are not doing it just because we have so much stuff that does not work on Windows yet...